metrics/cli/lib/prompt/strings/gcloud_strings.dart

```diff
@@ -0,0 +1,10 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+/// A class that holds the strings for the GCloud prompts.
+class GcloudStrings {
```

Comment:
I believe we should move this class to be under the `lib/gcloud/strings` package.